packages/filesystem/src/node/encoding-util.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Written by Xuye Cai <a1994846931931@gmail.com>, June 2019
```

Comment:
use our Copyright template, only update name of an entity and the year, please review new files in the PR

Reply:
Does the concept of "entity" refer to a natural person as well? I changed them to `Copyright (C) 2019 Xuye Cai and others.` Is it OK?

Reply:
yes, it is the standard Eclipse template